src/layers/StaticImageLayer.js

```diff
@@ -41,7 +41,7 @@ function scaleBounds({ width, height, translate, scale }) {
  * buffer, but without digging deeper into the WebGL it is a reasonable fix.
  */
 function padEven(data, width, height) {
-  const targetWidth = (width * height) % 2 === 0 ? width : width + 1;
```

Comment:
hmm I was getting the issue in other examples when it was the combination that wasn't even, not any individual dimension. Why the change here?

Reply:
I was getting the opposite!  Agh! Let me do some digging.

Reply:
@manzt If the width is even, so is the combination so I think we're safe.  If you have any specific examples where this doesn't work, you can slack me the link or post it here.

Reply:
I suppose if width were not even but the width x height were even, we would end up padding in this new check but not in the old.  But making it more even probably shouldn't hurt?